pgmq-extension/sql/pgmq--1.4.4--1.5.0.sql

```diff
@@ -0,0 +1,98 @@
+-- read
+-- reads a number of messages from a queue, setting a visibility timeout on them
+DROP FUNCTION pgmq.read(TEXT, INTEGER, INTEGER);
+CREATE OR REPLACE FUNCTION pgmq.read(
+    queue_name TEXT,
+    vt INTEGER,
+    qty INTEGER,
+    conditional JSONB DEFAULT '{}'
+)
+RETURNS SETOF pgmq.message_record AS $$
+DECLARE
+    sql TEXT;
+    qtable TEXT := pgmq.format_table_name(queue_name, 'q');
+BEGIN
+    sql := FORMAT(
+        $QUERY$
+        WITH cte AS
+        (
+            SELECT msg_id
+            FROM pgmq.%I
+            WHERE vt <= clock_timestamp() AND message @> %I
```

Comment:
Looks like this is in the right direction. Although I'm not sure `%I` will work for jsonb type, it might need to be a `%L`?

https://github.com/tembo-io/pgmq/actions/runs/11430681669/job/31799398848?pr=322

Also, if message is `null` then this will make `pgmq.read()` unable to return any messages since the DEFAULT value of the `conditional` parameter is `'{}'`

Reply:
pgmq.read() no longer fails when I change that `%I` to a `%L`, but it is unable to retrieve when message value is `null`.

```
postgres=# select pgmq.create('test');
 create 
--------
 
(1 row)

postgres=# select pgmq.send('test', null);
 send 
------
    1
(1 row)

postgres=# select pgmq.read('test', 1, 1);
 read 
------
(0 rows)

postgres=# select * from pgmq.q_test;
 msg_id | read_ct |          enqueued_at          |             vt              | message 
--------+---------+-------------------------------+-----------------------------+---------
      1 |       0 | 2024-10-20 20:56:14.760672-04 | 2024-10-20 20:56:14.7619-04 | 
(1 row)
```

Reply:
@ChuckHend Should be fixed now.